tasks/clamav-scan.yaml

```
@@ -41,7 +41,12 @@ spec:
         [ -f /workspace/registry-auth/.dockerconfigjson ] && REGISTRY_ARGS="-a /workspace/registry-auth/.dockerconfigjson"
         mkdir content
```

```suggestion
        mkdir content
```
create only if it doesn't exists

---

This folder is initiating place where the image will be extracted, so it will be created each time as new(it does not exist before each run).